crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs

```diff
@@ -1537,19 +1537,71 @@ impl Analyzer<'_, '_> {
     pub(crate) fn expand_intrinsic_types(&mut self, span: Span, ty: &Intrinsic) -> VResult<Type> {
         let arg = &ty.type_args;
 
-        match ty.kind {
-            IntrinsicKind::Uppercase | IntrinsicKind::Lowercase | IntrinsicKind::Capitalize | IntrinsicKind::Uncapitalize => {
```

Comment:
Since there is no `non_exhaustive` attribute, the first match statement may not be necessary. What do you think?
Or if this PR doesn't fix it, I'll roll it back.
Or attach the `non_exhaustive` macro to `IntrinsicKind` and express it as an error?

Reply:
I think `#[non_exhaustive]` is not required